modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -8,6 +8,7 @@ class AutoEstablishedClaim < ApplicationRecord
     attr_encrypted(:form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
     attr_encrypted(:auth_headers, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
     attr_encrypted(:evss_response, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
+    attr_encrypted(:bgs_responses, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
```

Comment:
Knowing that this is for flashes specifically and that we will be querying this to know when the flashes job passes or fails and that we will soon be working on special issues in the same fashion, does it make sense to store seperate BGS responses for flashes and special issues? Being that it's an encrypted cell we won't really be able to query for when flashes vs special issues errors exist etc.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/5335#discussion_r526219909

Reply:
> I went ahead with a more specific column name. if we're strongly considering needing a boolean for each flash and special_issues failures, then might as well keep the responses themselves separate.

https://github.com/department-of-veterans-affairs/vets-api/pull/5335#discussion_r526559134